src/UIComponents/DataCard.jsx

```diff
@@ -23,57 +24,58 @@ class DataCard extends Component {
     const dataLengthLimit = 20000;
     if (dataLength > dataLengthLimit) {
       window.alert(
-        "Warning: Datasets with more than 20,000 rows will slow down the user experience."
+        I18n.t("dataCardWarningLargeDataset", {"rowCount": dataLengthLimit})
       );
     }
 
     const removedRowsMsg =
       removedRowsCount > 0
-        ? removedRowsCount === 1
-          ? `${this.props.removedRowsCount} row was removed because it contained an empty cell.`
-          : `${this.props.removedRowsCount} rows were removed because they contained empty cells.`
+        ? I18n.t("dataCardRemovedRows", {"rowCount": removedRowsCount})
```

Comment:
We lose the "1 row" case?

Reply:
I am curious how crowdin would handle the pluralization cases... because you would normally do something like (Rails i18n in my example ... [messageformat has a different take](https://messageformat.github.io/messageformat/guide/#pluralformat)):

```
dataCardRemovedRows:
  one: "{count} row removed because it contained empty cells.",
  other: "{count} rows removed because they contained empty cells.",
```

And then `I18n.t("dataCardRemovedRows", {"count": removedRowsCount})`

Reply:
You are right @wilkie, I tried to do that but this repo is using an old version of `messageformat` which doesn't have that API. When I tried to upgrade `messageformat` I got some errors I didn't know how to deal with so I did a quick workaround. I will work on updating `messageformat` and doing what you suggested.

Reply:
The package has been updated to support plurals.